README.md

```diff
@@ -1,14 +1,61 @@
-# XAITK - Saliency
+![xaitk-logo](./docs/figures/xaitk-wordmark.png)
 
-## Intent
-Provide interfaces that convey a standard API for visual saliency
-map generation.
+<hr/>
+
+[![Documentation Status](https://readthedocs.org/projects/xaitk-saliency/badge/?version=latest)](https://xaitk-saliency.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Can we also add badges for the current pypi version (v0.4.0) and also python version (e.g. python 3.6/7/8)? Surveying other XAI packages, these seem to be included in their READMEs. I'm OK removing the "quality gate" and "code quality" badges if we need more space.

Reply:
I've removed the "Quality Gate Status" and added pypi latest-release version and supported python versions badges. Much more than this seems likely superfluous for now.

Reply:
Sounds good to me- thanks!